### PR TITLE
Add(route): add 深圳市委组织部 & 深圳考试院

### PR DESCRIPTION
**Pull request by zlasd**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/gov/shenzhen/zzb/tzgg
/gov/shenzhen/zzb/gzdt
/gov/shenzhen/zzb/zcfg
/gov/shenzhen/zzb/xcbd
/gov/shenzhen/zzb/bgxz
/gov/shenzhen/hrss/szksy/tzgg
/gov/shenzhen/hrss/szksy/bmxx
/gov/shenzhen/hrss/szksy/cjxx
/gov/shenzhen/hrss/szksy/hgbz
/gov/shenzhen/hrss/szksy/hgrygs
/gov/shenzhen/hrss/szksy/zsff
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [x] New Route
- [x] Documentation
  - [x] CN
  - [ ] EN
- [ ] 全文获取 fulltext
  - [ ] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] 日期和时间 date and time
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/gov/shenzhen/hrss/szksy.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+const rootURL = 'http://hrss.sz.gov.cn/';
+
+module.exports = async (ctx) => {
+    const categoryID = ctx.params.caty;
+    const page = ctx.params.page || '1';
```
Comment (TonyRL) [inline lib/routes/gov/shenzhen/hrss/szksy.js:8]:
```suggestion
    const page = ctx.params.page ?? '1';
```

File: lib/routes/gov/shenzhen/zzb/zzb.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+const rootURL = 'http://www.zzb.sz.gov.cn/';
+
+module.exports = async (ctx) => {
+    const categoryID = ctx.params.caty;
+    const page = ctx.params.page || '1';
```
Comment (TonyRL) [inline lib/routes/gov/shenzhen/zzb/zzb.js:8]:
```suggestion
    const page = ctx.params.page ?? '1';
```

File: lib/routes/gov/shenzhen/hrss/szksy.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    const title = $('.zx_rm_tit span').text().trim();
+    const list = $('.zx_ml_list ul li')
+        .slice(1)
+        .map((_, item) => {
+            const tag = $(item).find('div.list_name a');
+            const tag2 = $(item).find('span:eq(1)');
+            return {
+                title: tag.text().trim(),
+                link: tag.attr('href'),
+                pubDate: new Date(tag2.text().trim()).toUTCString(),
```
Comment (TonyRL) [inline lib/routes/gov/shenzhen/hrss/szksy.js:29]: Please use `parseDate()`
docs: https://docs.rsshub.app/joinus/pub-date.html

File: lib/routes/gov/shenzhen/zzb/zzb.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    const $ = cheerio.load(response.data);
+    const title = $('#Title').text().trim();
+    const list = $('#List tbody tr td table tbody tr td[width="96%"]')
+        .map((_, item) => {
+            const tag = $(item).find('font a');
+            const tag2 = $(item).find('font[size="2px"]');
+            return {
+                title: tag.text(),
+                link: tag.attr('href'),
+                pubDate: new Date(tag2.text().trim()).toUTCString(),
```
Comment (TonyRL) [inline lib/routes/gov/shenzhen/zzb/zzb.js:28]: Please use `parseDate()`
docs: https://docs.rsshub.app/joinus/pub-date.html

**Review by TonyRL — CHANGES_REQUESTED** [review]
Please use v2 script standard https://github.com/DIYgod/RSSHub/discussions/8236

**Comment by zlasd** [comment]
> Please use v2 script standard #8236

thanks for reviewing, now the router rule is up-to-date

File: docs/government.md [hunk 5]
```diff
@@ -212,6 +212,28 @@ pageClass: routes
...
+### 深圳市考试院
+
+<Route author="zlasd" example="/shenzhengov/szksy/bmxx/2" path="/shenzhengov/szksy/:caty/:page?" :paramsDesc="['信息类别', '页码']">
+
+| 通知公告 | 报名信息 | 成绩信息 | 合格标准 | 合格人员公示 | 证书发放信息 |
+| :------: | :------: | :------: | :------: | :----------: | :----------: |
+|   tzgg   |   bmxx   |   cjxx   |   hgbz   |    hgrygs    |     zsff     |
+
+</Route>
+
```
Comment (TonyRL) [inline docs/government.md:236]: Please put it in this section

https://github.com/DIYgod/RSSHub/blob/b36b326742e18f1f1506c3a7c59c565dfb6f1b82/docs/government.md#L61-L91

**Review by TonyRL — CHANGES_REQUESTED** [review]

**Comment by TonyRL** [comment]
I think you can also keep your previous route structures 
```
/gov/shenzhen/zzb/tzgg
/gov/shenzhen/hrss/szksy/tzgg
```

**Comment by zlasd** [comment]
> I think you can also keep your previous route structures
> 
> ```
> /gov/shenzhen/zzb/tzgg
> /gov/shenzhen/hrss/szksy/tzgg
> ```

I will change it back later. In the beginning, I'm afraid that it would conflict with v1 router rule.

**Comment by zlasd** [comment]
> I will change it back later. In the beginning, I'm afraid that it would conflict with v1 router rule.

done

**Review by TonyRL — APPROVED** [review]

**Pull request merged**